pages.es/netbsd/df.md

```diff
@@ -0,0 +1,32 @@
+# df
+
+> Muestra una visión general del uso del espacio en disco del sistema de archivos.
+> Más información: <https://man.netbsd.org/NetBSD-9.3/df.1>.
```

Comment:
I think this link should be replaced in all pages (although in another pull request) because it is outdated, I propose using `https://man.netbsd.org` as it currently supports searching manuals for the current version.